apps/platform-app/src/app/[locale]/(profile)/(components)/LocaleSwitcher/LocaleSwitcher.tsx

```diff
@@ -0,0 +1,26 @@
+import { I18nNamespaces } from '@/i18n'
+import { useLocale, useTranslations } from 'next-intl'
+import { LocaleSwitcherSelect } from './LocaleSwitcherSelect'
+
+// export const LocaleSwitcher = () => {
```

Comment:
**nitpick**: Consider removing commented-out code if it's no longer needed, or add a comment explaining why it's kept.